File: 10 - Python/Exercícios/03 - Estruturas Repetitivas/While/Exercício 04.py
#Exercício - Quadrante

print("Digite os valores das coordenas x e y:")
x = float(input(""))
y = float(input(""))

while x != 0 and y != 0:
    if x > 0 and y > 0:
        print("Quadrante 1")
    elif x < 0 and y > 0:
        print("Quadrante 2")
    elif x < 0 and y < 0:
        print("Quadrante 3")
    else:
        print("Quadrante 4")
    print("")
    print("Digite os valores das coordenas x e y:")
    x = float(input(""))
    y = float(input(""))

File: 07 - C++/Exercícios/03 - Estruturas Repetitivas/While/Exercício 04.cpp
//Exercício - Quadrante

#include <iostream>

using namespace std;

int main() {

    double x, y;

    cout << "Digite os valores das coordenadas X e Y:\n";
    cin >> x;
    cin >> y;

    while (x != 0 && y != 0){
        
        if(x > 0 && y > 0){
            cout << "Quadrante Q1\n";
        }
        else if(x < 0 && y > 0){
            cout << "Quadrante Q2\n";
        }
        else if(x < 0 && y < 0){
            cout << "Quadrante Q3\n";
        }
        else{
            cout << "Quadrante Q4\n";
        }

        cout << "Digite os valores das coordenadas X e Y:\n";
        cin >> x;
        cin >> y;
    
    }
    
    return 0;

}